generators/php/codegen/src/context/PhpAttributeMapper.ts

```diff
@@ -103,6 +111,8 @@ export class PhpAttributeMapper {
                     writer.write("::class");
                 });
             }
+            case "enumString":
+                return php.codeblock('"string"');
```

Comment:
Should we prefer single quotes here?
```suggestion
                return php.codeblock("'string'");
```